### PR TITLE
chore: use cz-git commitizen adapter

### PR DESCRIPTION
**Pull request by Zhengqbbb**
Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

Hi, the author of my cz-git commitizen adapter, I noticed that your project uses strict commit message specification, I think cz-git can greatly improve your development experience

- github: https://github.com/Zhengqbbb/cz-git
- website: https://cz-git.qbenben.com/
- 中文文档：https://cz-git.qbenben.com/zh/

#### Feature:
- **Friendly**: Supports search and selection on the command line, reducing spelling errors and repetitive input.
- **With Commitlint**: Dynamically get commitlint configuration and give command line prompts.
- **Lightweight** There is only one cz-git dependency(1.7 MB)

#### Demo

demo

- Get the git status information, and put the matching scope information (packages/*) pin to the top.

demo

- Get the git status information and display the matching component information. You can complate through <kbd>tab</kbd> or <kbd>→</kbd>

demo

#### By the way 
I removed the package from `@pnpm/find-workspace-packages` in commitlint, I found that it would take 300+ ms to use it

image

Hope it can help you well, And if you have any questions, can tell me in time. Thx ~

### CODE REVIEW

**Comment by ryuhangyeong** [comment]
Wow~
=

**Comment by Zhengqbbb** [comment]
I noticed that you will use `pnpm cz ` to start commitizen, and cz-git does not include commitizen dependencies
I am not sure if this is used properly, but it will undoubtedly increase the overall dependency of the project, but I still Keep this way of use
If you need to remove it, can tell me in time

File: commitlint.config.js [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
-  const ctx = context || {}
-  const cwd = ctx.cwd || process.cwd()
-  const packages = await getWorkspacePackages(cwd)
-  return packages
-    .map((pkg) => pkg.manifest.name)
-    .filter((name) => !!name)
-    .map((name) => (name.charAt(0) === '@' ? name.split('/')[1] : name))
-}
+const getPackages = (packagePath) =>
+  fs
```
Comment (sxzz) [inline commitlint.config.js:7]: IMHO, we can use `fast-glob` here to simplify code.
```ts
fg.sync("*", { cwd: packagePath, onlyDirectories: true })
```

Comment (Zhengqbbb) [reply commitlint.config.js]: > IMHO, we can use `fast-glob` here to simplify code.
> 
> ```ts
> fg.sync("*", { cwd: packagePath, onlyDirectories: true })
> ```

Great !!! have done

**Review by sxzz — APPROVED** [review]
LGTM

**Pull request merged**